config/settings.yml

```diff
@@ -353,6 +353,7 @@ modules_appeals_api:
       # See features.yml for enablement
       recipients:
         - drew.fisher@adhocteam.us
+        - matt.kelly@adhocteam.us
```

Comment:
I didn't realize these `recipients` were still here! They're no longer used, since the recipients list is pulled from the YAML files now. (They used to be pulled from Settings.) You can go ahead and remove L353 - L358.
We might also want to add a line for `enabled: false` under `weekly_error` to match the rest of the reports.